tests/integration/test_enrollment/test_agent/test_agent_auth_enrollment.py

```diff
@@ -157,7 +158,7 @@ def test_agent_auth_enrollment(test_configuration, test_metadata, set_wazuh_conf
 
         try:
             # Start socket monitoring
-            socket_monitor.start(timeout=10, accumulations=2, callback=lambda received_event: received_event.encode() in event)
+            socket_monitor.start(timeout=60 if sys.platform == WINDOWS else 20, accumulations=2, callback=lambda received_event: received_event.encode() in event)
```

Comment:
I would like to understand a timeout as an "impossible situation", not a performance problem. Therefore, I would prefer to avoid relying on the platform, if there is no compelling reason.

Reply:
This was modified in order to unify with the other enrollment test for windows:
https://github.com/wazuh/wazuh/blob/917794389d2946763a2189cc32e3214cd595830f/tests/integration/test_enrollment/test_agent/test_agentd_enrollment.py#L156